vm/proxyapp/proxyappclient.go

```diff
@@ -172,7 +171,7 @@ func runProxyApp(params *proxyAppParams, cmd string) (*ProxyApp, error) {
 	onTerminated := make(chan bool, 1)
 
 	go func() {
-		io.Copy(os.Stdout, subprocessLogs)
+		io.Copy(params.LogOutput, subprocessLogs)
```

Comment:
There are 2 implementations: production and test. Test does not set LogOutput (I assume tests don't exercise this code). So it's only for production and it always uses os.Stdout. So why do we need this?

Reply:
https://github.com/google/syzkaller/pull/3395 use it in tests.